processing/src/main/java/org/apache/carbondata/processing/loading/model/LoadOption.java

```diff
@@ -59,7 +59,8 @@
     optionsFinal.put("fileheader", Maps.getOrDefault(options, "fileheader", ""));
     optionsFinal.put("commentchar", Maps.getOrDefault(options, "commentchar", "#"));
     optionsFinal.put("columndict", Maps.getOrDefault(options, "columndict", null));
-
+    optionsFinal.put("sort_scope",
+        Maps.getOrDefault(options, "sort_scope", CarbonCommonConstants.LOAD_SORT_SCOPE_DEFAULT));
```

Comment:
Please check in the CrabonSession Load scenario we are restricting it or not. It is not supposed to use per load . It should be only for SDK

Reply:
yes, it should not be set here.

now moved to 
CarbonLoadModelBuilder.build() , so it will set only for SDK and spark-file-format